src/views/End/End.vue

```diff
@@ -71,24 +71,38 @@ export default {
       const { hash } = this.$route
       const hashUrl = new URL(`${window.location.origin}?${hash.slice(1)}`)
       const result = hashUrl.searchParams.get('result')
+
       let whiteLabel = {}
       let loginConfig = {}
 
       let loginError = ''
 
+      let resultParams = {
+        store: {},
+      }
+      const sessionId = hashUrl.searchParams.get('sessionId') || ''
+      const sessionNamespace = hashUrl.searchParams.get('sessionNamespace') || ''
       if (result) {
-        const resultParams = JSON.parse(safeatob(result))
+        resultParams = JSON.parse(safeatob(result))
         loginError = resultParams.error
         const appStateParams = JSON.parse(safeatob(resultParams.store.appState))
-        whiteLabel = appStateParams.whiteLabel || {}
-        loginConfig = appStateParams.loginConfig || {}
+        const appSessionData = appStateParams[getIFrameOrigin()] || appStateParams
```

Comment:
cannot use getIframeOrigin here.. cause it'll be in popup

Reply:
it is not required now, removed.